backend/app/models/activity.rb

```diff
@@ -24,4 +24,6 @@ class Activity < Content
 
   has_many :related_contents
   has_many :publications, through: :related_contents
+
+  acts_as_taggable
```

Comment:
This is defined in their parent "Content", so doesn't need to be declared here again.